docs/backward-compat.md

```diff
@@ -0,0 +1,18 @@
+# Backward Compatibility
+
+Backup taken by Velero vSphere plugin with version lower or equal to 1.0.2 can be restored by Velero vSphere plugin version 1.1.0
+only on a Vanilla Setup.
+
+Prior to attempting restore or delete backup (<=1.0.2), ensure that a vSphere snapshot location exists prior to attempting a restore of older backup.
+If not, create one.
+
+```bash
+velero snapshot-location create <snapshot location name> --provider velero.io/vsphere
+```
+
+After restore/delete of <=1.0.2 plugin Backup, the vSphere snapshot location can be safely deleted. The vSphere snapshot
+location is only necessary when attempting to restore or delete a older(<=1.0.2) plugin backup. 
```

Comment:
a older(<=1.0.2) plugin backup -> a backup from an older plugin (<=1.0.2)
Same comment applies to other places as well.